README.md

```diff
@@ -1 +1,22 @@
 # action-assert-dh-tables-present
+
+This action validates the presence of Deephaven tables within a workflow.
+
+## Parameters
+
+| Parameter | Description | Required |
+|--|--|--|
+| table-names | A string containing a comma separated list of table names to check. | Yes |
+| host | The host name of the Deephaven instance. Defaults to `envoy`. | No |
+| project-name | The docker-compose project name to bundle separate docker-compose files | Yes |
+
+## Example
+
+```
+- name: Validate tables
+  uses: deephaven/action-assert-dh-tables-present@main
+  with:
+    table-names: "source,result"
+    host: "envoy"
```

Comment:
I don't think this needs quoting

Reply:
Does the same either way, I'll remove the quotes since that's a little cleaner